app/controllers/concerns/foreman/controller/parameters/nic_base.rb

```diff
@@ -24,6 +23,7 @@ def add_nic_base_params_filter(filter)
         :username,
         :virtual,
         {:attached_devices => []},
+        {:compute_attributes => [:bridge, :from_profile, :model, :network, :type, :name]},
```

Comment:
> > compute_attributes is generally passed through without filtering
> 
> I saw that, but thought the whole point of strong parameters was filtering.

Yes, I was just unsure if it was feasible. I forgot that this only needs to list network interface attributes, not the VM-level ones, so it's OK - please disregard this.